app/views/courses/TeacherClassView.coffee

```diff
@@ -171,7 +177,8 @@ module.exports = class TeacherClassView extends RootView
     null
 
   onLoaded: ->
-    @sortedCourses = @classroom.getSortedCourses()
+    @latestCourses = @courses.where({releasePhase: 'released'})
+    utils.sortCourses(@latestCourses)
```

Comment:
Because of this same reason, this function does nothing and should be updated: https://github.com/codecombat/codecombat/blame/classroom-autoupdate/server/models/Course.coffee#L22